src/signals/incident-management/containers/StandardTextsAdmin/provider.tsx

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+/* istanbul ignore file */
+import type { FC, ReactNode } from 'react'
+
+import { StandardTextsAdminContext } from './context'
+import type { StandardTextsAdminValue } from './types'
+
+export const initialValue: StandardTextsAdminValue = {
+  page: 1,
+  setPage: () => {},
+}
+
+interface Props {
+  children: ReactNode
+  value: StandardTextsAdminValue
+}
+
+export const StandardTextsAdminProvider: FC<Props> = ({ value, children }) => {
```

Comment:
This FC Props is something we don't do anymore right?, just children}: Props can be done as well. At least thats what I thought we agreed on